src/renderer/components/+workloads-deployments/deployment-scale-dialog.scss

```diff
@@ -20,11 +20,17 @@
     }
 
     .desired-scale {
-      flex: 1 0;
+      flex: 1.1 0;
     }
 
     .slider-container {
-      flex: 1.3 0;
+      flex: 1 0;
+    }
+
+    .plus-minus-container {
+      margin-left: 15px;
+      // for vertical align with .slider-container
+      margin-top: -4px;
     }
```

Comment:
Also, please add this lines inside `.plus-minus-container`
```
.Icon {
  --color-active: black;
}
```
This will fix `:hover` and `:active` colors for the icon.

Reply:
ah, thanks, the weird `:hover` and `:active` coloring is the reason I am not using `<Icon />`